src/dbus_stats.rs

```diff
@@ -0,0 +1,342 @@
+//! # dbus_stats module
+//!
+//! Handle getting statistics of our Dbus daemon/broker
+
+use std::sync::Arc;
+use std::collections::HashMap;
+
+use anyhow::Result;
+use tokio::sync::RwLock;
+use tracing::error;
+use zbus::fdo::StatsProxy;
+use zvariant::{OwnedValue, Value, Dict};
+
+use crate::MachineStats;
+
+// Unfortunatelly, various DBus daemons (ex: dbus-broker and dbus-daemon)
+// represent stats differently. Moreover, the stats vary across versions of the same daemon.
+// Hence, the code uses flexible approach providing max available information.
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Default, Eq, PartialEq)]
+pub struct DBusBrokerPeerAccounting {
+    pub name: String,
+
+    // credentials
+    pub unix_user_id: Option<u32>,
+    pub process_id: Option<u32>,
+    pub unix_group_ids: Option<Vec<u32>>,
+    // ignoring LinuxSecurityLabel
+    // pub linux_security_label: Option<String>,
```

Comment:
Let make note of this in the issue that someone else can implement this if needed

Reply:
do you want me to create an issue?